src/sentry/tasks/store.py

```diff
@@ -279,6 +281,13 @@ def _continue_to_process_event():
 
     symbolication_start_time = time()
 
+    submission_ratio = options.get("symbolicate-event.low-priority.metrics.submission-rate")
+    submit_realtime_metrics = not from_reprocessing and random.random() < submission_ratio
+
+    if submit_realtime_metrics:
+        with sentry_sdk.start_span(op="tasks.store.symbolicate_event.low_priority.metrics.counter"):
+            realtime_metrics.increment_project_event_counter(project_id, symbolication_start_time)
```

Comment:
Since this part is not critical for the symbolication itself, should we prevent the task from crashing in case something goes wrong with the `realtime_metrics.increment_project_event_counter`?

Reply:
In my latest commit I added a `try` block; is that sufficient to make the task not crash?